### PR TITLE
Faceook -> Facebook typos

### PR DESCRIPTION
**Pull request by petercpg**

### CODE REVIEW

**Comment by flodolo** [comment]
I'm going to merge this one, because I need to see what happens in Pontoon when a message changes, without a new ID.

File: src/_locales/en/messages.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
       "description": "This is alternative text for the back arrow button."
     },
     "btn-arrow-forward": {
-      "message": "Go to How Faceook Container Works",
+      "message": "Go to How Facebook Container Works",
```
Comment (flodolo) [inline src/_locales/en/messages.json]: This is the change I neeed to look out for (string is translated in zh-TW)
https://pontoon.mozilla.org/zh-TW/facebook-container/all-resources/?search=btn-arrow-forward&string=198587

**Comment by flodolo** [comment]
> I'm going to merge this one, because I need to see what happens in Pontoon when a message changes, without a new ID.

Works as expected. The existing translations remain there, I'll double check if someone else copied they typo in their translation.

**Comment by flodolo** [comment]
>  I'll double check if someone else copied they typo in their translation.

Fixed ka, sq in Pontoon :-\

**Comment by pdehaan** [comment]
Also spotted these, but they're only descriptions and nothing user facing, so not sure if you want me to bother with a PR:

```sh
$ git grep -i Face src/_locales | grep -vi "Facebook"

src/_locales/cy/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/cy/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/de/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/de/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/en-CA/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/en-CA/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/es-ES/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/fr/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/fr/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/fy-NL/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/pt-BR/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/pt-BR/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/pt-PT/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/pt-PT/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/sv-SE/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/vi/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/vi/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/zh-CN/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/zh-CN/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/zh-TW/messages.json:    "description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
src/_locales/zh-TW/messages.json:    "description": "This is the second paragraph of the second onboarding panel. Please do not translate \"Faceook Container\""
```

**Comment by flodolo** [comment]
No, Pontoon will fix them the first time it commits something for those locales.

**Pull request merged**